fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/Expression.java

```diff
@@ -18,18 +18,22 @@
 package org.apache.doris.nereids.trees.expressions;
 
 import org.apache.doris.nereids.exceptions.UnboundException;
-import org.apache.doris.nereids.rules.expression.rewrite.ExpressionVisitor;
 import org.apache.doris.nereids.trees.AbstractTreeNode;
 import org.apache.doris.nereids.trees.NodeType;
 import org.apache.doris.nereids.types.DataType;
 
+import org.slf4j.Logger;
```

Comment:
use interface is more flexible and reasonable then a concrete implement

Reply:
> use interface is more flexible and reasonable then a concrete implement

yes, but fe logger system has some bug, slf4j maybe print out log info into fe.out instead of fe.log.
maybe we need to fix it later😂

Reply:
😂, ok